cfpb/cfgov-refresh
# Fix for Google Map image of Washington DC on /archive-past-events/ page.

Google changed the location image for [Washington](https://maps.googleapis.com/maps/api/staticmap?center=Washington&size=276x155&zoom=12&scale=2). This PR fixes that by adding the the venue state field to the parameter.


## Changes

- Modified [http://localhost:8000/about-us/events/archive-past-events/](http://localhost:8000/about-us/events/archive-past-events/) to add the venue state field to the location parameter. 

## Testing

- Visit `http://localhost:8000/about-us/events/archive-past-events/`.
- Verify that the correct images are showing for the cities.

## Screenshots
<img width="949" alt="screen shot 2017-01-27 at 9 37 26 am" src="https://cloud.githubusercontent.com/assets/1696212/22374456/438478e2-e474-11e6-9b2b-b94ce6298c0c.png">


## Checklist
